ansible_collections/arista/avd/molecule/eos_validate_state/reports/FABRIC-state.csv

```diff
@@ -1,1711 +1,1711 @@
+41,dc1-leaf1a,Connectivity,VerifyReachability,Test the network reachability to one or many destination IP(s).,Source: P2P Interface Ethernet6 (IP: 10.255.255.0) - Destination: dc1-wan1 Ethernet1 (IP: 10.255.255.1),NOT RUN,
+42,dc1-leaf1a,Connectivity,VerifyReachability,Test the network reachability to one or many destination IP(s).,Source: P2P Interface Ethernet7 (IP: 10.255.255.4) - Destination: dc1-wan2 Ethernet1 (IP: 10.255.255.5),NOT RUN,
+43,dc1-leaf1a,Field Notices;Software,VerifyFieldNotice44Resolution,Verifies the device is using an Aboot version that fix the bug discussed in the field notice 44 (Aboot manages system settings prior to EOS initialization),,NOT RUN,
+44,dc1-leaf1a,Field Notices;Software,VerifyFieldNotice72Resolution,"Verifies if the device has exposeure to FN72, and if the issue has been mitigated",,NOT RUN,
+45,dc1-leaf1a,Hardware,VerifyAdverseDrops,Verifies there are no adverse drops on DCS7280E and DCS7500E,,NOT RUN,
+46,dc1-leaf1a,Hardware,VerifyEnvironmentCooling,Verifies the status of power supply fans and all fan trays.,,NOT RUN,
+47,dc1-leaf1a,Hardware,VerifyEnvironmentCooling,Verifies the status of power supply fans and all fan trays.,Accepted States: 'ok',NOT RUN,
+48,dc1-leaf1a,Hardware,VerifyEnvironmentPower,Verifies the power supplies status.,,NOT RUN,
+49,dc1-leaf1a,Hardware,VerifyEnvironmentPower,Verifies the power supplies status.,Accepted States: 'ok',NOT RUN,
+50,dc1-leaf1a,Hardware,VerifyEnvironmentSystemCooling,Verifies the system cooling status.,,NOT RUN,
+51,dc1-leaf1a,Hardware,VerifyTemperature,Verifies the device temperature.,,NOT RUN,
+52,dc1-leaf1a,Hardware,VerifyTemperature,Verifies the device temperature.,,NOT RUN,
+53,dc1-leaf1a,Hardware,VerifyTransceiversManufacturers,Verifies if all transceivers come from approved manufacturers.,,NOT RUN,
+54,dc1-leaf1a,Hardware,VerifyTransceiversManufacturers,Verifies if all transceivers come from approved manufacturers.,"Accepted Manufacturers: 'Arista Networks', 'Arastra, Inc.', 'Not Present'",NOT RUN,
+55,dc1-leaf1a,Hardware,VerifyTransceiversTemperature,Verifies the transceivers temperature.,,NOT RUN,
+56,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Ethernet1 - P2P_LINK_TO_DC1-SPINE1_Ethernet1 = 'up',NOT RUN,
+57,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Ethernet2 - P2P_LINK_TO_DC1-SPINE2_Ethernet1 = 'up',NOT RUN,
+58,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Ethernet3 - MLAG_PEER_dc1-leaf1b_Ethernet3 = 'up',NOT RUN,
+59,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Ethernet4 - MLAG_PEER_dc1-leaf1b_Ethernet4 = 'up',NOT RUN,
+60,dc1-leaf1a,Interfaces,VerifyInterfacesStatus,Verifies the status of the provided interfaces.,Interface Ethernet5 - dc1-leaf1-server1_PCI1 = 'up',NOT RUN,
```

Comment:
Same as other can we use capital latters for consistency

Reply:
This is generated from molecule. That means the hosts are in lowercase which is totally valid.